fhir-config/src/main/java/com/ibm/fhir/config/FHIRConfiguration.java

```diff
@@ -109,6 +109,10 @@
     public static final String PROPERTY_NATS_KEYSTORE = "fhirServer/notifications/nats/keystoreLocation";
     public static final String PROPERTY_NATS_KEYSTORE_PW = "fhirServer/notifications/nats/keystorePassword";
 
+    // Operations config properties
+    public static final String PROPERTY_OPERATIONS_EVERYTHING = "fhirServer/operations/everything";
```

Comment:
ok, this one is my bad... I should have said to also add the new config property to the User's Guide (section 5).
specifically there are 3 tables there and we add to each one (our attempt to avoid table width limitations on the UI).
we could probably merge this one and then add the doc in a follow-up PR

Reply:
Done